kiagnose/internal/config/environment_test.go

```diff
@@ -27,19 +27,18 @@ import (
 	"github.com/kiagnose/kiagnose/kiagnose/internal/config"
 )
 
-func TestConfigMapFullNameShouldSucceed(t *testing.T) {
+func TestCheckupFullNameShouldSucceed(t *testing.T) {
```

Comment:
nit: @qinqon WDYT about:
```suggestion
TestCheckupKeyFromEnvShouldSucceed(t *testing.T) {
```
?

Reply:
Thank you.